platforms/osx/src/main.mm

```diff
@@ -13,11 +13,9 @@
 
 std::string sceneFile = "scene.yaml";
 
-std::string markerStyling = "{ style: 'points', interactive: true, color: 'white', size: [25px, 25px], order: 100, collide: false }";
+std::string markerStyling = "touch.point.icons";
```

Comment:
I did reply to your earlier comment. Its implicit in the naming of the API that it is a `layer` path, this is much different than the usage of the scene update interface where the user must identity with the complete yaml path.

That said I have addressed this in a270ba951bd4a38e3ff605ff01461811a14aa563. Thanks.